lib/streamlit/config.py

```diff
@@ -408,6 +408,24 @@ def _server_cookie_secret():
     return cookie_secret
 
 
+@_create_option("server.enableCORS", type_=bool)
+def _server_enable_cors():
+    """Enables support for Cross-Origin Request Sharing, for added security.
+
+    Default: true
+    """
+    return True
```

Comment:
This seems backwards to me. In `routes.py` we check if CORS is allowed by using `not config.get_option("server.enableCORS")`. Left it as is for now 🤷‍♀️ 
```
`def allow_cross_origin_requests():
    """True if cross-origin requests are allowed.

    We only allow cross-origin requests when using the Node server. This is
    only needed when using the Node server anyway, since in that case we
    have a dev port and the prod port, which count as two origins.

    """
    return not config.get_option("server.enableCORS") or config.get_option(
        "global.useNode"
    )
```

Reply:
I can do that. I was being silly and slowly alphabetizing them.